client/src/elements/schema-editor/schema-editor-files.js

```diff
@@ -191,7 +209,12 @@ export class SchemaEditorFiles {
         this.dropzone.files.push(mockFile);
         this.dropzone.emit("addedfile", mockFile);
         if (file.type && file.type.includes("image/")) {
-          mockFile.dataURL = file.url; // needed for dropzone to create the thumbnail in a canvas
+          const imageUrl = fileBaseUrl ? `${fileBaseUrl}/${file.key}` : file.url;
+          const response = await this.httpClient.fetch(imageUrl);
+          if (!response.ok || response.status >= 400) {
```

Comment:
> The ok read-only property of the Response interface contains a Boolean stating whether the response was successful (status in the range 200-299) or not.

There's no need for the `response.status` condition

Reply:
true